organization/src/main/kotlin/com/cosmotech/organization/azure/OrganizationServiceImpl.kt

```diff
@@ -278,7 +278,10 @@ class OrganizationServiceImpl(private val csmRbac: CsmRbac, private val csmAdmin
   ): OrganizationAccessControl {
     val organization = findOrganizationById(organizationId)
     csmRbac.verify(organization.getRbac(), PERMISSION_WRITE_SECURITY)
-    csmRbac.getAccessControl(organization.getRbac(), identityId)
+    csmRbac.getAccessControlOrThrow(
```

Comment:
Following the dev review on cosmotech-api-common, use the correct method `(verify|check)Rights`

Reply:
done